docs/docs/build/contracts/index.md

```diff
@@ -0,0 +1,36 @@
+# Contracts
+
+## BOB Testnet (Sepolia)
+
+### Multicall3
+
+- Multicall3: `0x089b191d95417817389c8eD9075b51a38ca46DE8`
+
+### BOB Core Contracts
+
+- tBTC Relay: `0x077c5ed60fABb260784891786c6573373fDa8A3E`
```

Comment:
@gregdhill can you confirm this is the correct contract that I deployed? https://github.com/bob-collective/bob/blob/mt-and-aa2/src/relay/DummyRelay.sol

Reply:
We need to deploy the `TestLightRelay` on the master branch.

Reply:
fixed